src/css/app.scss

```diff
@@ -24,6 +24,12 @@ body {
     padding-right: 10px;
 }
 
+/* Fix grid title rendering on Chrome.
```

Comment:
fix comment

Reply:
I think it's worth putting in Browser versions in the comments also, so if the issue resolves itself down the road with a newer release there's an end point.